packages/client/src/features/validation/validate.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2019-2023 EclipseSource and others.
+ * Copyright (c) 2019-2024 EclipseSource and others.
```

Comment:
We should try to avoid non-relevant changes like these.

Reply:
This get autofixed if you use the `check:headers` command. Seems like we missed to update the header in https://github.com/eclipse-glsp/glsp-client/pull/329

Reply:
Got it, thanks!